demo-site/templates/test_endpoints/test_endpoints_app.js

```diff
@@ -0,0 +1,94 @@
+const path = require('path');
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const { env } = require('process');
+const { loadJson } = require("../../utils/utils");
+const { 
+    TEST_APP_CREDENTIALS_ENDPOINT,
+    SUPPORTED_FEATURES_ENDPOINT,
+    CONFIG_ENDPOINT,
+    PX_METADATA_FILE_NAME
+} = require('../../utils/constants');
+
+class TestEndpointsApp {
```

Comment:
Not necessary since the test endpoints are included in the `servers/nodejs/app.js` server